src/tests/test_certstor_macos.cpp

```diff
@@ -0,0 +1,336 @@
+/*
+* (C) 1999-2019 Jack Lloyd
+* (C) 2019      René Meusel
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include "tests.h"
+
+#include <botan/build.h>
+
+#include <iostream>
+
+#if defined(BOTAN_TARGET_OS_IS_DARWIN)
```

Comment:
This should be checking for `BOTAN_HAS_CERTSTOR_MACOS` since it is possible to build on OSX but with the certstore disabled.